files/jira.rb

```diff
@@ -5,10 +5,16 @@
 class Jira < BaseHandler
 
   def build_labels
-    [ "SENSU_#{@event['client']['name']}",
+    user_labels = [*(@event['check']['tags'] || [])]
```

Comment:
team_data('tags') is something we control. I also checked one of sensu-server.log files - check tags are either an empty list, a list of strings or nil.